install.md

```diff
@@ -72,6 +72,8 @@ sudo apt-get install -qq -y software-properties-common uidmap
 sudo add-apt-repository -y ppa:projectatomic/ppa
 sudo apt-get update -qq
 sudo apt-get -qq -y install podman
+sudo mkdir -p /etc/containers
+echo -e "[registries.search]\nregistries = ['docker.io']" | sudo tee /etc/containers/registries.conf
```

Comment:
Could you add quay.io as well, since we don't want to say we support only one registry by default.